functions.php

```diff
@@ -594,7 +594,7 @@ function zw_rest_api_init()
                         ];
                         break;
 
-                    case 'blok_dossier':
+                    /* case 'blok_dossier':
```

Comment:
Waarom is dit uitgecomment?

Reply:
Maakt app kapot. Die snapt dit blok nog niet